common/src/View/GamesPreferencePane.cpp

```diff
@@ -80,8 +80,10 @@ namespace TrenchBroom {
             auto* container = new QWidget();
 
             m_gamePathText = new QLineEdit();
-            m_gamePathText->setReadOnly(true);
             setHint(m_gamePathText, "Click on the button to change...");
+            connect(m_gamePathText, &QLineEdit::editingFinished, this, [this]() {
```

Comment:
This works of course, but now you can easily set non existing directories and it doesn't indicate any problem with that. I think we should show some sort of warning when the directory does not exist.

Reply:
K, I'll add some kind of validator

Reply:
I just added a warning icon when the directory doesn't exist:

![Capture](https://user-images.githubusercontent.com/239161/79008469-b8527800-7b1a-11ea-8685-1969de53bd1d.PNG)